### PR TITLE
Fix for the flaky Horizon Test

### PR DESCRIPTION
**Pull request by Meuge**
There was an error with the epoch.
This addresses the issue #968

### CODE REVIEW

File: tests/tests_twobody/test_orbit.py [hunk 1]
```diff
@@ -1034,21 +1034,17 @@ def test_from_coord_if_coord_is_not_of_shape_zero():
 def test_from_sbdb_and_from_horizons_give_similar_results(target_name):
     ss_target = Orbit.from_sbdb(target_name)
     ss_classical = ss_target.classical()
-
-    ss_ref = Orbit.from_horizons(
-        name=target_name, attractor=Sun, plane=Planes.EARTH_ECLIPTIC
-    )
-
-    ss_ref = ss_ref.propagate_to_anomaly(
```
Comment (jorgepiloto) [inline tests/tests_twobody/test_orbit.py:1042]: Which was the aim of propagating the orbit in the old version? I mean, the method returns the orbit with that specific true anomaly, isn't it?

Comment (Meuge) [reply tests/tests_twobody/test_orbit.py:1042]: What I understood, this test was to double check that the data from sddb and from horizon gave the same results. In order to do so, I send the same epoch in both methods and the result was positive.

File: tests/tests_twobody/test_orbit.py [hunk 2]
```diff
@@ -629,8 +629,8 @@ def test_orbit_from_horizons_has_expected_elements():
     )
     ss1 = Orbit.from_horizons(name="Ceres", attractor=Sun, epoch=epoch)
     assert ss.pqw()[0].value.all() == ss1.pqw()[0].value.all()
-    assert ss.r_a == ss1.r_a
-    assert ss.a == ss1.a
+    assert_quantity_allclose(ss.r_a, ss1.r_a, rtol=1.0e-4)
+    assert_quantity_allclose(ss.a, ss1.a, rtol=1.0e-4)
```
Comment (jorgepiloto) [inline tests/tests_twobody/test_orbit.py:633]: This assertion looks good! Good job @Meuge :rocket:

File: tests/tests_twobody/test_orbit.py (hunk 1 above)
Comment (astrojuanlu) [reply tests/tests_twobody/test_orbit.py:1042]: > What I understood, this test was to double check that the data from sddb and from horizon gave the same results. In order to do so, I send the same epoch in both methods and the result was positive.

Yes, that was the idea of #867!

**Pull request merged**